script.service.checkpreviousepisode/addon.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.service.checkpreviousepisode" name="Kodi Check Previous Episode" provider-name="bossanova808, Razzeee, Lucleonhart" version="0.4.1">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0" />
+    <import addon="script.module.yaml" version="3.11.0" />
+	<import addon="xbmc.addon" version="18.9.701" />
+  </requires>
+  <extension library="default.py" point="xbmc.service" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">Kodi Check Previous Episode</summary>
+    <summary lang="bg_BG">Изгледан ли е предходния епизод</summary>
+    <description lang="en_GB">This service helps prevent spoilers by checking if the previous episode in a series has been watched. If not, it will pause playback and warn you.  Specific shows can be marked to be ignored if episode order does not matter.</description>
+    <description lang="de_DE">Dieses Script überprüft die vorige Episode der TV Show die gerade gestartet werden soll und warnt falls diese noch nicht gesehen wurde oder nicht vorhanden ist!</description>
+    <description lang="bg_BG">Когато пускате епизод от даден сериал скриптът проверява предходния и предупреждава ако той отсъства от библиотеката или не е изгледан.</description>
+    <platform>all</platform>
+    <language />
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```

Comment:
spdx if possible

Reply:
You can find the valid SPDX strings here https://spdx.org/licenses/